Server/XCTest+CBXAdditions.h

```diff
@@ -76,3 +77,17 @@
 - (XCUIHitPointResult *_Nullable)hitPoint:(int8_t *_Nullable)arg1;
 
 @end
+
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
I think we don't want to introduce these macros.  Is there a reason for these?

Reply:
Replaced to `_Nonnull`